scss/components/_responsive-embed.scss

```diff
@@ -50,10 +49,14 @@ $responsive-embed-ratio-widescreen: 16 by 9 !default;
 
 @mixin foundation-responsive-embed {
   .responsive-embed, .flex-video {
-    @include responsive-embed;
+    @include responsive-embed( map-get($responsive-embed-ratios,default) );
 
-    &.widescreen {
-      padding-bottom: responsive-embed($responsive-embed-ratio-widescreen);
+    @each $name, $ratio in $responsive-embed-ratios {
+      @if $name != default {
```

Comment:
I think the "map for modifiers with a default value that should be skipped" can be factorized with the breakpoint stuff :)

For example: [/scss/util/_mixins.scss:234](https://github.com/ncoden/foundation-sites/blob/6c8c184f6785478f4c70b1e7ab0bbac32c6ac3d0/scss/util/_mixins.scss#L234)